### ISSUE TITLE
[volspotconnect2 0.6.0] Can not connect from mobile app

### ISSUE THREAD

**Issue by messismore** [volumio/volumio-plugins#153]
Hi,
connecting from the macOS and Windows desktop apps works fine, but on iOS or Android it will say _connecting…_ for a while before it eventually fails. I had previously used the version from @balbuze's [repo](https://github.com/balbuze/volumio-plugins/tree/master/plugins/music_service/volspotconnect2) and it had been rock solid…

**Comment by messismore** [comment]
Not sure if this is related, but I just noticed that the player shows up twice in the Spotify app: 

<img width="323" alt="bildschirmfoto 2018-03-02 um 14 01 43" src="https://user-images.githubusercontent.com/5689189/36900245-d2799e72-1e22-11e8-984e-a42430d5c5bb.png">

The player's name is _Kueche_

**Comment by ashthespy** [comment]
https://github.com/balbuze/volumio-plugins/issues/133#issuecomment-369896611
Cheers! :-)

**Comment by messismore** [comment]
Thanks, I had missed that 😄 Do plugins installed via the UI update automatically? I would just wait it out then…

**Comment by ashthespy** [comment]
Yes and no - you should get an 'update' button when the plugin has been officially added to the plugin manager.

**Comment by dudedaoud** [comment]
Il have the same problem. No connections with Android

**Comment by ashthespy** [comment]
@dudedaoud  what version of the plugin are you running?

**Comment by dudedaoud** [comment]
The Last 18.02.2018 plugin

**Comment by messismore** [comment]
The plugin has still not been updated, unfortunately...

**Comment by balbuze** [comment]
You have to install the Dev version waiting the official update.

**Comment by dudedaoud** [comment]
THX but how can i download it an install ? my volumio dont have an option to install downloaded plugin.

NB : im french and i speak like a spanish cow.... 👍

**Comment by ashthespy** [comment]
@dudedaoud See my previous [comment](https://github.com/volumio/volumio-plugins/issues/153#issuecomment-369936898) ;-)
> balbuze#133 (comment)
> Cheers! :-)

**Comment by messismore** [comment]
Is there an ETA on this @volumio?

**Comment by volumio** [comment]
Yep, testing as we speak

**Comment by dudedaoud** [comment]
Great !! It work perfectly. Updated now

**Comment by dudedaoud** [comment]
Vous faites un super boulot. Merci à tous les mecs !!

**Comment by dudedaoud** [comment]
Hi. Dont work after update. Cant play on volumio by my Android app. Spotify updated his app against hack of apps i can see.... I have a premium account

**Comment by balbuze** [comment]
Really they did ? I see an update is available on my smartphone... Maybe I shouldn't update... I'm going to check with the current version.

**Comment by balbuze** [comment]
I updated spotify app to the last on my smartphone and it still working with the plugin.
